speculos/main.py

```diff
@@ -139,7 +140,22 @@ def get_cx_infos(app_path):
     return sh_offset, sh_size, sh_load, cx_ram_size, cx_ram_load
 
 
-def run_qemu(s1: socket.socket, s2: socket.socket, args: argparse.Namespace) -> int:
+def get_elf_fonts_size(app_path):
+    '''
+    Check for C_bagl_fonts symbol to determine if the app use BAGL or NBGL syscalls.
+    If C_bagl_fonts is found, it means that this app uses BAGL (useful on NanoX and NanoSP)
```

Comment:
This won't work for app based on the Rust SDK. We need to find something for them.

Reply:
Do you mean now and here in Speculos, or is it a job for another time or tool?

Reply:
Oh okay so it pretty much blocking at this point.

Reply:
though there are easy dirty fix like : detect it's a rust app => launch speculos to support BAGL syscall.

Reply:
Does the new code answers this issue?